content/en/tracing/trace_retention_and_ingestion/_index.md

```diff
@@ -72,11 +72,11 @@ To customize what spans are indexed and retained for 15 days, you can create, mo
 
 1. Name your filter.
 2. Set the tags you would like to index spans that match **all** of.
-3. Select whether this filter will retain any span that matches the criteria, or only [top level spans][4].
+3. Select whether this filter will retain any span that matches the criteria, or only [service entry spans][4].
 4. Set a percentage of spans matching these tags to be indexed.
 5. Save your new filter.
 
-**Note:** Selecting "Top-Level Spans for Services Only" means the retention filter will retain only the selected proportion of [top level spans][4] of service and index them. Use this if you want to only index top level spans with matching tags. If "All Spans" is selected, the retention filter will retain the selected proportion of all spans of the distributed trace, irrespective of their hierarchy, and index them.  This may have an impact on your bill, and the visual indicator within the app while setting a retention filter will inform you how many matching spans have been detected over the time period.
+**Note:** Selecting "Top-Level Spans for Services Only" means the retention filter will retain only the selected proportion of [service entry spans][4] of service and index them. Use this if you want to only index service entry spans with matching tags. If "All Spans" is selected, the retention filter will retain the selected proportion of all spans of the distributed trace, irrespective of their hierarchy, and index them.  This may have an impact on your bill, and the visual indicator within the app while setting a retention filter will inform you how many matching spans have been detected over the time period.
```

Comment:
```suggestion
**Note:** Selecting "Top-Level Spans for Services Only" means the retention filter retains only the selected proportion of [service entry spans][4] of service and index them. Use this if you want to only index service entry spans with matching tags. If "All Spans" is selected, the retention filter retains the selected proportion of all spans of the distributed trace, irrespective of their hierarchy, and index them.  This may have an impact on your bill, and the visual indicator within the app while setting a retention filter informs you how many matching spans have been detected over the time period.
```